portfolio/src/main/webapp/index.html

```diff
@@ -6,12 +6,18 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getMsg()">
+  <body onload="getComments()">
     <div id="name">
       <h1>George Zheng</h1>
     </div>
     <hr>
-    <div id="message-container"></div>
+    <ul id="comments-history"></ul>
+    <form action="/comments" method="POST">
+      <p>Add a new comment:</p>
+      <input type="text" name="comment" value="">
```

Comment:
I think the empty value attribute can be removed?

Reply:
Fixed.